ray-operator/controllers/ray/utils/util.go

```diff
@@ -359,3 +426,22 @@ func GenerateJsonHash(obj interface{}) (string, error) {
 
 	return hashStr, nil
 }
+
+// Equals returns true if the two lists are equivalent
+func Equals(a corev1.ResourceList, b corev1.ResourceList) bool {
```

Comment:
Let's use https://pkg.go.dev/k8s.io/apiserver/pkg/quota/v1#Equals?

Reply:
Good find!

Reply:
Hmm, actually it looks like pulling in this package broke a lot of things.

Reply:
Ah, okay, figured out which version we need to use. Should be good to use this library.